release/ray_release/test_automation/state_machine.py

```diff
@@ -207,6 +230,16 @@ def _close_github_issue(self) -> None:
         issue.edit(state="closed")
         self.test.pop(Test.KEY_GITHUB_ISSUE_NUMBER, None)
 
+    def _keep_github_issue_open(self) -> None:
+        github_issue_number = self.test.get(Test.KEY_GITHUB_ISSUE_NUMBER)
+        if not github_issue_number:
+            return
+        issue = self.ray_repo.get_issue(github_issue_number)
+        if issue.state == "open":
+            return
+        issue.create_comment("Re-opening issue as test is still failing.")
+        issue.edit(state="open")
```

Comment:
maybe open it first and then comment?

also, maybe comment with more information, like a link to the test job?